index.js

```diff
@@ -2369,6 +2382,17 @@
 
   //. ### Object
 
+  //# prop :: Accessible a => String -> a -> b
+  //.
+  //. Takes a property name and returns the value of that property from the
+  //. specified object; otherwise throws an error.
```

Comment:
How about this?

_Takes a property name and a record (an object with known fields of known types), and returns the value of the specified field. If for some reason the record lacks the specified field, a type error is thrown._

_For accessing properties of uncertain objects, use [`get`][1] instead._

We should add a "see also" link to the description of `get`.


[1]: https://github.com/plaid/sanctuary#get